Demo/Application/Features/Helpers/UIViewController+Extensions.swift

```diff
@@ -1,29 +0,0 @@
-import UIKit
-
-extension UIViewController {
```

Comment:
Weird this file is showing as deleted when it doesn't exist on the `payment-insights-feature` branch